gui/src/installer/view.rs

```diff
@@ -885,6 +888,128 @@ pub fn define_bitcoin<'a>(
     )
 }
 
+pub fn select_bitcoind_type(progress: (usize, usize), error: Option<&String>) -> Element<Message> {
+    layout(
+        progress,
+        "Choose Bitcoin installation type",
+        Column::new()
+            .push(text(prompt::SELECT_BITCOIND_TYPE))
+            .push(
+                Row::new()
+                    .align_items(Alignment::End)
+                    .spacing(20)
+                    .push(
+                        Container::new(
+                            Column::new()
+                                .spacing(20)
+                                .align_items(Alignment::Center)
+                                .push(
+                                    button::primary(None, "I want to manage my own node")
+                                        .width(Length::Fixed(300.0))
+                                        .on_press(Message::SelectBitcoindType(
+                                            message::SelectBitcoindTypeMsg::UseInternal(false),
+                                        )),
+                                )
+                                .align_items(Alignment::Center),
+                        )
+                        .padding(20),
+                    )
+                    .push(
+                        Container::new(
+                            Column::new()
+                                .spacing(20)
+                                .align_items(Alignment::Center)
+                                .push(
+                                    button::primary(None, "Let Liana manage my node")
+                                        .width(Length::Fixed(300.0))
+                                        .on_press(Message::SelectBitcoindType(
+                                            message::SelectBitcoindTypeMsg::UseInternal(true),
+                                        )),
+                                )
+                                .align_items(Alignment::Center),
+                        )
+                        .padding(20),
+                    ),
+            )
+            .push_maybe(error.map(|e| card::invalid(text(e)))),
+        true,
+    )
+}
```

Comment:
this looks better on my side
```suggestion
pub fn select_bitcoind_type(progress: (usize, usize), error: Option<&String>) -> Element<Message> {
    layout(
        progress,
        "Choose Bitcoin installation type",
        Column::new()
            .push(text(prompt::SELECT_BITCOIND_TYPE))
            .spacing(20)
            .push(
                Row::new()
                    .align_items(Alignment::End)
                    .spacing(20)
                    .push(
                        button::primary(None, "I want to manage my own node")
                            .width(Length::FillPortion(2))
                            .on_press(Message::SelectBitcoindType(
                                message::SelectBitcoindTypeMsg::UseInternal(false),
                            )),

                    )
                    .push(
                        button::primary(None, "Let Liana manage my node")
                            .width(Length::FillPortion(2))
                            .on_press(Message::SelectBitcoindType(
                                message::SelectBitcoindTypeMsg::UseInternal(true),
                            )),
                    ),
            )
            .push_maybe(error.map(|e| card::invalid(text(e)))),
        true,
    )
}
```

Reply:
Thanks for this. There may be other changes required on this page, such as more detailed explanations of the different options, so I'll wait for those before making these changes.